x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/InternalPreconfiguredEndpoints.java

```diff
@@ -97,6 +98,16 @@ public record MinimalModel(
                     ChunkingSettingsBuilder.DEFAULT_SETTINGS
                 ),
                 GP_LLM_V2_COMPLETION_SERVICE_SETTINGS
+            ),
+            new MinimalModel(
+                new ModelConfigurations(
+                    GP_LLM_V2_COMPLETION_ENDPOINT_ID,
+                    TaskType.COMPLETION,
+                    ElasticInferenceService.NAME,
+                    GP_LLM_V2_COMPLETION_SERVICE_SETTINGS,
+                    ChunkingSettingsBuilder.DEFAULT_SETTINGS
```

Comment:
I just realized we have a subtle bug here that I introduced initially. I tested and it doesn't seem to affect anything, thankfully. Could you try setting this to null?

There shouldn't be any chunking settings for completion or chat completion endpoints.